transformations/gender_neutral_rewrite/transformation.py

```diff
@@ -0,0 +1,482 @@
+    verb_tense = verb.morph.get("Tense")[0]  # list with 1 item
+
+    if verb_tense == "Past":
+        # was is an irregular past tense verb from third-person singular to third-person plural
+        if verb_text.lower() == "was":
+            return capitalization_helper(verb_text, "were")
+
+        # other past-tense verbs remain the same
+        else:
+            return None
+
+    # oftentimes, if there are 2+ verbs in a sentence, each verb after the first (the conjuncts) will be misclassified
+    # the POS of these other verbs are usually misclassified as NOUN
+    # e.g. He dances and prances and sings. --> "prances" and "sings" are conjuncts marked as NOUN (should be VERB)
+    # checking if verb ends with "s" is a band-aid fix
+    elif verb_tense == "Pres" or verb.text.endswith("s"):
+        return capitalization_helper(
+            original=verb_text.lower(),
+            replacement=pluralize_present_simple(verb_text),
+        )
+
+    return None
+
+
+def pluralize_present_simple(lowercase_verb: str):
+    """
+    pluralize a third-person singular verb in the present simple tense
+    :param lowercase_verb: original verb (lower-cased)
+    :return: 3rd-person plural verb in the present simple tense
+    """
+    for singular, plural in IRREGULAR_VERBS.items():
+        if lowercase_verb == singular:
+            return plural
+
+    if lowercase_verb.endswith("ies"):
+        return lowercase_verb[:-3] + "y"
+
+    # -es rule: https://howtospell.co.uk/adding-es-plural-rule
+    for suffix in VERB_ES_SUFFIXES:
+        if lowercase_verb.endswith(suffix):
+            return lowercase_verb[:-2]
+
+    if lowercase_verb.endswith("s"):
+        return lowercase_verb[:-1]
+
+    return None
+
+
+def create_new_doc(doc: Doc, verbs_replacements: dict):
+    """
+    create a new SpaCy doc using the original doc and a mapping of verbs to their replacements
+    :param doc: original doc with simple_replace extension (from simple_replace function)
+    :param verbs_replacements: dictionary mapping verbs and auxiliaries to their replacements
+    :return: the gender-neutral sentence as a str
+    """
+    token_texts = []
+    for token in doc:
+        replace_verb = (
+            token in verbs_replacements.keys() and verbs_replacements[token]
+        )
+
+        if token._.simple_replace:
+            token_texts.append(token._.simple_replace)
+
+        elif replace_verb:
+            token_texts.append(verbs_replacements[token])
+
+        else:
+            token_texts.append(token.text)
+        if token.whitespace_:  # filter out empty strings
+            token_texts.append(token.whitespace_)
+
+    new_sentence = "".join(token_texts)
+    return new_sentence
+
+
+class GenderNeutralRewrite(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+        TaskType.TEXT_TAGGING,
+    ]
+    languages = ["en"]
+
```

Comment:
Missing the [keywords](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/docs/keywords.md) variable here.

Reply:
Added.